docker-compose.yml

```diff
@@ -97,6 +97,23 @@ services:
     labels:
       be.vliz.container.project: "LWUA"
       be.vliz.container.group: "services"
-      be.vliz.container.owner: "laurian.van.maledghem@vliz.be"
+      be.vliz.container.owner: "laurian.van.maldeghem@vliz.be"
       be.vliz.container.description: "LifeWatch User-Analysis Platform"
-      be.vliz.container.vcs-url: "https://github.com/lifewatch/user-analysis-2023.git"
\ No newline at end of file
+      be.vliz.container.vcs-url: "https://github.com/lifewatch/user-analysis-2023.git"
+
+  sembench:
+    build:
+      context: ./docker/lwua-sembench/
+    image: lwua/lwua_sembench
+    container_name: lwua_sembench
+    volumes:
+      - ./data:/data
+      - ./config:/config
+    environment:
+      - INPUT_DATA_LOCATION=/data/input-data
+      - OUTPUT_DATA_LOCATION=/data/output-data
```

Comment:
think input and data will just sit together throughout the /data folder 
so both to just point to `/data` 

also best to indicate what this actually does --> I assume the paths in config/sembench.json can refer to these, or are assumed relative to these?

Reply:
those paths are considered as a base path in the sembench.json yes